components/OverhangCard/OverhangCard.styles.ts

```diff
@@ -1,22 +1,34 @@
 import { cnb } from 'cnbuilder';
 import { type ImageAspectRatioType } from '@/components/Image';
-import { type CardBgColorType, cardBgColors } from '@/utilities/datasource';
+import { allCardBgColors, type AllCardBgColorType } from '@/utilities/datasource';
 
 export type OverhangCardVariantType = 'basic' | 'tile' | 'quote' | 'story';
 
 export const root = (
+  variant: OverhangCardVariantType,
   isVertical: boolean,
+  isFeatured: boolean,
+  hasLink: boolean,
   largeCardPadding: boolean,
-  bgColor: CardBgColorType,
+  bgColor: AllCardBgColorType,
 ) => cnb(
-  'relative break-words shadow-md border border-black-10',
-  cardBgColors[bgColor], {
-    'md:mt-80 px-32 pb-32 md:h-[calc(100%_-_8rem)]': isVertical,
-    'md:ml-80 pr-32 py-32': !isVertical,
+  'group relative break-words shadow-md border border-black-10',
+  hasLink && 'focus-within:shadow-lg hover:shadow-lg transition-shadow',
+  allCardBgColors[bgColor],
+  {
+    'flex-col': isVertical,
+    'flex-row': !isVertical && variant === 'basic',
+    'flex-row-reverse justify-between': !isVertical && variant !== 'basic',
+    'md:mt-80 px-32 pb-32 md:h-[calc(100%_-_8rem)]': isVertical && variant === 'basic',
+    'md:ml-80 pr-32 py-32': !isVertical && variant === 'basic',
+    'w-4/5 sm:w-3/4 md:w-full mx-auto': variant === 'story' && !isFeatured,
+    'mt-80 px-32 pb-32 !h-[calc(100%_-_8rem)]': variant === 'story' && isVertical,
+    'lg:mr-[20vw] 2xl:mr-300 lg:pl-48 rs-pb-3': variant === 'story' && !isVertical,
     'md:px-72 md:pb-72 2xl:px-78 2xl:pb-78': largeCardPadding && isVertical,
     'md:px-36 md:pb-36 2xl:px-38 2xl:pb-38': !largeCardPadding && isVertical,
-    'md:pr-72 md:py-72 2xl:pr-78 2xl:py-78': largeCardPadding && !isVertical,
-    'md:pr-36 md:py-36 2xl:pr-38 2xl:py-38': !largeCardPadding && !isVertical,
+    'md:pr-72 md:py-72 2xl:pr-78 2xl:py-78': largeCardPadding && !isVertical && variant !== 'story',
+    'md:pr-36 md:py-36 2xl:pr-38 2xl:py-38': !largeCardPadding && !isVertical && variant !== 'story',
+    'md:px-36 md:pb-45 2xl:px-38 2xl:pb-48': variant === 'story' && isVertical,
```

Comment:
This can be cleaned up further - I'll do that after I get the tile and quote cards in